packages/fields/src/fields/author/author-view.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import clsx from 'clsx';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useState } from '@wordpress/element';
+import { commentAuthorAvatar as authorIcon } from '@wordpress/icons';
+import { __experimentalHStack as HStack, Icon } from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+import { store as coreStore } from '@wordpress/core-data';
+import type { User } from '@wordpress/core-data';
+
+/**
+ * Internal dependencies
+ */
+import type { BasePostWithEmbeddedAuthor } from '../../types';
+
+function AuthorView( { item }: { item: BasePostWithEmbeddedAuthor } ) {
+	const { text, imageUrl } = useSelect(
+		( select ) => {
+			const { getEntityRecord } = select( coreStore );
+			let user: User | undefined;
+			if ( !! item.author ) {
+				user = getEntityRecord( 'root', 'user', item.author );
+			}
```

Comment:
There's an opportunity for improvement in this field. This code is triggering a request for user data (avatar + username), and the Pages page may potentially trigger as many requests as pages are visible if they have different authors.

But we already have that data via multiple sources:

- The `usePostFields` hook requests all the users.
- The request to the `pages` endpoint embeds the author data in the item. However, this is done is a way that is not there from the beginning, but rather the pages data come in and then later it comes the `_embedded` piece of it. This prevents us from using the `_embedded` data directly and only trigger the request if it becomes stale (the user updated the item's author via QuickEdit).

It's all a bit too convoluted, and there's no obvious solution, so I'd suggest looking at this separately from this PR.

Thoughts about potential ways forward:

1. Given the item has `elements` that are the "valid elements", we could use that as a source for data retrieval. It means augmenting the elements with miscelaneous data plus making them accessible to the `render` function. By combining this with optimizing the data request so it is only received by DataViews when the `embedded` data is added to the entity would allow us to eliminate all author REST requests.
2. Finding a way to augment item data. This is related to the conversation at https://github.com/WordPress/gutenberg/pull/65426#discussion_r1838573793